src/components/CityProfileCard.js

```diff
@@ -70,42 +110,50 @@ class CityProfileCard extends Component {
     }
 
     return (
-      <div>
-        <Slide direction="right" in={this.state.checked} mountOnEnter unmountOnExit>
-          <Paper className={classes.paper}>
-            <div className="headerContainer">
-              <img src={require('../' + cityData.headerImage)} alt={`${cityData.cityName} Header`}
-                   style={{ width: '100%', height: '100%', filter: 'brightness(60%)' }}/>
-              <div className="alignIconHeader">
-                <RankingIcon cityData={cityData}/>
-                <h1 className="cityHeader">{cityData.cityName}</h1>
+      <Hammer key={cityData.key} onSwipe={this.handleSwipe}>
+        <div>
+          <Slide direction="right" in={this.state.checked} mountOnEnter unmountOnExit>
+            <Paper className={classes.paper}>
+              <div className="headerContainer">
+                {prevCityData != null ? 
```

Comment:
This could probably just be `prevCityData` without the `!= null`. Also, we probably would want a double equals here if we kept it.